pyanaconda/modules/security/certificates/installation.py

```diff
@@ -38,6 +41,24 @@ def __init__(self, sysroot, certificates):
     def name(self):
         return "Import CA certificates"
 
+    def _dump_certificate(self, cert, root):
+        """Dump the certificate into specified file and directory."""
+        dst_dir = join_paths(root, cert.dir)
+        if not os.path.exists(dst_dir):
+            log.debug("Path %s for certificate %s does not exist, creating.",
+                      dst_dir, cert.filename)
+            mkdirChain(dst_dir)
+
+        dst = join_paths(dst_dir, cert.filename)
```

Comment:
What if the file already exists ? While it will most likely write over it, it might be a good idea to log that we are overwriting an existing file.

Reply:
Good catch, we even have this upstream, I must have somehow missed the patch:
https://github.com/rhinstaller/anaconda/commit/649c6bb4b8f2806a095bc8f098a62315d89a4b01